### DISCUSSION TITLE
Using autogen assistants to talk over a dataset. But when im querying the dataset, assistantproxy agent is not able to filter the dataset according to the question and giving random records which are not part of my dataset  [Bug]:

### DISCUSSION THREAD

**Discussion by PrashanthReddy1823** [microsoft/autogen#2576] [Q&A]
### Describe the bug

Hi team, Im using autogen multiple agents. Im passing a csv file while initiating the chat. But when im passing a message related to dataset i have passed its not giving correct answers and also display records those are not part of my dataset at all. How can we validate the output given by  assistant agent? 

### Model Used

gpt-3.5-turbo

### Expected Behavior

I need to work agent accurately according the user questions

### Additional Information

using pyautogen==0.2.27.
I can provide the code snippets of required.

**Comment by ekzhu** [comment]
Your agents are hallucinating. Can you post your code?
